.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -15,16 +15,19 @@ on:
 env:
   BUILD_ENVIRONMENT: !{{ build_environment }}
   BUILD_WHEEL: 1
-  CUDA_VERSION: "cpu"
+  CUDA_VERSION: "!{{ cuda_version }}"
   IN_CI: 1
   INSTALL_WINDOWS_SDK: 1
   JOB_BASE_NAME: test
   PYTHON_VERSION: "3.6"
   SCCACHE_BUCKET: "ossci-compiler-cache"
-  TORCH_CUDA_ARCH_LIST: "5.2;7.5"
   VC_PRODUCT: "BuildTools"
   VC_VERSION: ""
   VC_YEAR: "2019"
+{%- if cuda_version != "cpu" %}
+  TORCH_CUDA_ARCH_LIST: "7.0"
```

Comment:
just wondering what's our support matrix for CUDA arches on windows? just sm_70?